add-on/chrome/modules/MozLoopService.jsm

```diff
@@ -866,12 +866,16 @@ var MozLoopServiceInternal = {
     return "about:loopconversation#" + chatWindowId;
   },
 
+  getLoopChatWindows: function() {
```

Comment:
Just 'getChatWindows` should do fine, I think, since you're working in loop code already.